cmd/deploy/deploy.go

```diff
@@ -0,0 +1,173 @@
+package deploy
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/google/uuid"
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/rest"
+)
+
+var tempKubeConfig = "/var/tmp/okteto.kubeconfig"
```

Comment:
I would use a file in the okteto home folder

Reply:
Could this be a const?